### PR TITLE
Playlist management

### PR DESCRIPTION
**Pull request by Pratik1005**
Added playlist management which have features like:

- add/delete playlist
- add/delete video to/from playlist

### CODE REVIEW

File: src/reducer/userDataReducer.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    case USER_ACTIONS.ADD_NEW_PLAYLIST:
+    case USER_ACTIONS.DELETE_PLAYLIST:
+      return {...state, playlist: [...action.payload]};
+    case USER_ACTIONS.ADD_VIDEO_TO_PLAYLIST:
+    // return {
+    //   ...state,
+    //   playlist: state.playlist.map((item) =>
+    //     item.title === action.payload.title ? action.payload : item
+    //   ),
+    // };
```
Comment (manismk) [inline src/reducer/userDataReducer.js]: remove the commented code if not needed

File: src/App.css [hunk 2]
```diff
@@ -7,6 +7,16 @@ ul {
   list-style-type: none;
 }
 
+.flex-center {
+  display: flex;
+  align-items: center;
+  gap: 10px;
```
Comment (shraddha15-creator) [inline src/App.css:13]: you can use rems here

**Review by manismk — APPROVED** [review]
Good naming convention and code is easy to understand

suggestion: Clear the create playlist input box after the playlist is created

**Review by shraddha15-creator — APPROVED** [review]
Looks good, Code structure good and readable as well.

**Pull request merged**